src/main/java/emu/grasscutter/data/excels/TalkConfigData.java

```diff
@@ -0,0 +1,45 @@
+package emu.grasscutter.data.excels;
+
+import com.google.gson.annotations.SerializedName;
+import emu.grasscutter.data.GameResource;
+import emu.grasscutter.data.ResourceType;
+import emu.grasscutter.game.talk.TalkExec;
+
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.experimental.FieldDefaults;
+
+import java.util.List;
+
+@ResourceType(name = "TalkExcelConfigData.json")
+@EqualsAndHashCode(callSuper=false)
+@Data
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class TalkConfigData extends GameResource {
+    @SerializedName(value="id", alternate={"_id"})
+    int id;
+    @SerializedName(value="finishExec", alternate={"_finishExec"})
+    List<TalkExecParam> finishExec;
+    @SerializedName(value="questId", alternate={"_questId"})
+    int questId;
```

Comment:
maybe you could also add the fields that TalkData uses here, and then mostly replace talk data with this.